### DISCUSSION TITLE
Will it also work for the playoffs?

### DISCUSSION THREAD

**Discussion by XiangjunWu** [kyleskom/NBA-Machine-Learning-Sports-Betting#156] [General]
The regular season is ending soon, will this tool work for the playoffs as well?

**Comment by kyleskom** [comment]
It should

**Reply by onlyadamscott** [reply]
It is so far. Doing pretty well. Running 71.8% ML model.
Question as I'm no ML or NN expert. Once trained, should I be moving the best trained models into "Models\XGBoost_Models"? Awesome work. I've gotten it to run in a handful of environments.

**Reply by onlyadamscott** [reply]
I sorted it out. Fantastically clean code. Well done.

**Reply by XiangjunWu** [reply]
Were you able to use your new trained model? Whenever I replace the old model in the file XGBoost_Runner.py with the new model. I receive an error after running the program.

**Reply by dexinthecity** [reply]
how do you use new trained models?

**Reply by kyleskom** [reply]
It already uses the most up to date models.

**Comment by JP23000** [comment]
14-6 so far this playoffs on OU.  skipped the play in games.

**Reply by onlyadamscott** [reply]
What are your models trained at? I'm at 71% or so.

**Reply by XiangjunWu** [reply]
Same achieved great results on OU. The prediction on OU works pretty well for the playoffs. It's because the teams are playing more seriously?
